chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -301,6 +300,12 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
 
         // Start chat from SDK Event
         BroadcastService.getMessageByEventName(BroadcastEvent.StartChat).subscribe((msg: ICustomEvent) => {
+            if (!isNullOrUndefined(msg?.payload?.runtimeId) &&
+                msg?.payload?.runtimeId !== TelemetryManager.InternalTelemetryData.lcwRuntimeId && // If the startChat event is not initiated by the same tab. Ignore the call
+                props.controlProps?.hideStartChatButton !== true) { // Skip the above check for popout chat
+                return;
+            }
```

Comment:
Just wondering, in case Chat is started using one tab and if user has refreshed the tab, the runtime id will change. What will happen in this case ?

Reply:
If the refresh happens before the chat starts, the new runtime id will be used in startChat. If it's after chat starts, refreshing the page will not invoke startChat again since the existing chat will rehydrate